.claude/skills/create-pr/SKILL.md

```
@@ -0,0 +1,49 @@
+disable-model-invocation: true
+---
+
+# PR 설명 작성
+
+`.github/pull_request_template.md`에 정의된 형식을 그대로 따른다.
+
+## 제목
+
+대괄호 태그로 시작한다 (커밋 type과 다른 별도 목록):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)